make/lint.mk

```diff
@@ -6,10 +6,15 @@ YAML_FILES := $(shell find . -type f -regex ".*y[a]ml" -print)
 .PHONY: lint-yaml
 ## runs yamllint on all yaml files
 lint-yaml: ${YAML_FILES}
+ifeq (, $(shell which yamllint))
```

Comment:
Is it something from another PR?

Reply:
yeah, same as above, not sure what happened here 😬 , but it's outdated now